src/components/button/button.css

```diff
@@ -60,3 +62,9 @@
 .btn-lg {
   font-size: 1.5rem;
 }
+
+.btn-float {
+  position: fixed;
+  bottom: 2rem;
+  right: 2rem;
+}
```

Comment:
Use descriptive classnames that matches the property. 
.btn-float feels something related to float. 
You can may be use  .btn-fixed,   .btn-absolute and so on.

Reply:
okay